services/prometheus/alerting_rules_test.go

```diff
@@ -53,14 +53,6 @@ groups:
 			rules := strings.TrimSpace(`
 groups:
 - name: example
-rules:
-- alert: HighRequestLatency
-expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
-for: 10m
-labels:
-severity: page
-annotations:
-summary: High request latency
```

Comment:
No, that's an issue with updated Prometheus. Please update error message in that test

Reply:
rename to FormerZero